### DISCUSSION TITLE
Is there a notice or timeline/roadmap for when support for v11 will end?

### DISCUSSION THREAD

**Discussion by jax7778** [felddy/foundryvtt-docker#1072] [Q&A] [answered]
Hello, 

I apologize for my ignorance, but is there any info on when support for Foundry v11 will end? I just don't want to be  blindsided. I could not find anything on it? 

Thanks in advance for any info.

**Comment by felddy** [comment]
I don't think I've seen the `closing` release of `v11` go out yet.  I would expect that you'd see that announced in the release notes of a `v11` release.  The best place to seek clarification is the official Discord.  

See:
- https://foundryvtt.com/article/versioning/
- https://foundryvtt.com/releases/
- https://discord.gg/foundryvtt

**Reply by jax7778** [reply]
Sorry, to clarify, when this project will no longer support running Foundry v11, not support from Foundry itself. I am trying to discover when this project will end support for running v11?

**Reply by felddy** [reply, marked as answer]
Thank you for the clarification.  

tl;dr: I wouldn't be concerned.  

The images for the container that supports `v11` will continue to persist even after active support for `v11` ends.  For example, `v10` container are still available on Docker Hub: https://hub.docker.com/r/felddy/foundryvtt/tags?page=2&name=10.

Historically there have been very few breaking changes between versions.  Even major versions.  Given that, you can take the current container that supports `v12` and request that it install a `v11` FoundryVTT release using the `FOUNDRY_VERSION` environment variable.

The most likely failure modes are when either the API (webpage) to request a FoundryVTT release is changed, or the `options.json` file needed by a release is significantly different.  In both cases there are work-arounds.  e.g., downloading release tarballs manually, or disabling configuration generation by the container.  

If we get into a state when the container isn't working for `v11` let me know and we'll figure out a solution.

**Reply by jax7778** [reply]
Thank you for the information. I really appreciate it.
